src/Umbraco.Infrastructure/Sync/DatabaseServerMessenger.cs

```diff
@@ -526,11 +533,7 @@ private void SaveLastSynced(int id)
         /// <para>Practically, all we really need is the guid, the other infos are here for information
         /// and debugging purposes.</para>
         /// </remarks>
-        protected string LocalIdentity => NetworkHelper.MachineName // eg DOMAIN\SERVER
-            + "/" + _hostingEnvironment.ApplicationId // eg /LM/S3SVC/11/ROOT
-            + " [P" + Process.GetCurrentProcess().Id // eg 1234
-            + "/D" + AppDomain.CurrentDomain.Id // eg 22
-            + "] " + Guid.NewGuid().ToString("N").ToUpper(); // make it truly unique
+        protected string LocalIdentity { get; }
```

Comment:
I think you can do this inline initialization if you wanted to, with `{ get; } = NetworkHelper...`.  That would do what I think you were intending from the inline method call.  It would do the same as moving it to the constructor, but would keep the calculation near the property for reference.